### DISCUSSION TITLE
How to get splines from an object

### DISCUSSION THREAD

**Discussion by fenjalien** [JacquesLucke/animation_nodes#1856] [Q&A]
Hi, I can't seem to get a spline from a curve object. I've only just picked up AN and I'm not sure if this is a bug or I'm doing something wrong.

Creating a new path curve (Add > Curve > Path) and using the object in the `Splines from Object` node yields no splines in the output, as shown in the viewer node. The object does have a mesh however. ![image](https://user-images.githubusercontent.com/34489450/167396108-eff630cd-3ba1-4415-b3e4-0723297f97a5.png)

Creating a spline using the vertex locations of the mesh adding and it to a new object does then show the new spline in the viewer (It also does become visible in the viewport).
![image](https://user-images.githubusercontent.com/34489450/167397380-f99ebe90-6b93-46a9-bbee-4d6376250345.png)

If this is a bug at least there is a workaround of generating the spline from the mesh vertex positions. Although it does lead to a spline of 48 points from an original of 5.

Windows 10
AN Version: 2.3 (latest)
Blender: 3.1.2

**Comment by OmarEmaraDev** [comment]
Animation Nodes doesn't support NURBs unfortunately, only bezier and and poly curves. Path curves are nurbs not bezier splines.
